oi/OI.py

```diff
@@ -11,4 +15,25 @@ def init() -> None:
     @staticmethod
     def map_controls():
         logger.info("Mapping controls...")
-        pass
+        
+        def intake_deploy():
+            Robot.intake.set_intake_angle(math.radians(config.intake_deploy_pos))
+            Robot.intake.set_roller_velocity(config.intake_roller_speed)
+
+        def intake_idle():
+            Robot.intake.set_intake_angle(math.radians(config.intake_standard_pos))
+            Robot.intake.set_roller_velocity(config.intake_idle_speed)
+
+        def intake_rotate(speed):
+            Robot.intake.rotate_intake(speed)
+
+        def zero_intake():
+            Robot.intake.zero_intake()
+
+        Keymap.Intake.DEPLOY_INTAKE.whileTrue(InstantCommand(intake_deploy)).onFalse(InstantCommand(intake_idle))
+
+        Keymap.Intake.ZERO_INTAKE.onTrue(zero_intake)
```

Comment:
We will zero the intake at the beginning of teleop or auto. Don't assign this functionality to a button.